src/main/java/net/sf/marineapi/nmea/sentence/UBXSentence.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2020 Gunnar Hillert
+ *
+ * This file is part of Java Marine API.
+ * <http://ktuukkan.github.io/marine-api/>
+ *
+ * Java Marine API is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * Java Marine API is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Java Marine API. If not, see <http://www.gnu.org/licenses/>.
+ */
+package net.sf.marineapi.nmea.sentence;
+
+import net.sf.marineapi.nmea.parser.SentenceParser;
+import net.sf.marineapi.nmea.sentence.Sentence;
+
+/**
+ *
```

Comment:
Please add some description in each comment, as the javadocs are the only "user interface" of the library, so empty comments look a bit incomplete. For example, in the methods it's a good practice to mention the unit of measure etc. to be explicit, although it often feels a bit repetitive to write "get <measure>, in <unit>".

Reply:
Added some more JavaDoc.